.github/workflows/py-hypertrace.yml

```diff
@@ -1,91 +0,0 @@
-name: Hypertrace
```

Comment:
These tests have been moved to `isofit/test/test_hypertrace.py`, labeled with a `pytest` marker, and are executed as part of the `pip` package test workflow. They can be selected with:

```console
$ pytest -m hypertrace
```